content/posts/newsletter-september-2024.md

```diff
@@ -0,0 +1,39 @@
+title: Read the Docs Newsletter - September 2024
+date: September 13, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/september-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@routessansfins?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Les routes sans fin(s)</a> on <a href="https://unsplash.com/photos/a-lifeguard-tower-sitting-on-top-of-a-sandy-beach-gnkhKUmENms?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+## News and updates
```

Comment:
Do we want to mention SAML in beta testing again so we can get more customers here and promote it a little more?